src/NuGet.Core/NuGet.Common/RuntimeEnvironmentHelper.cs

```diff
@@ -59,7 +59,7 @@ public static bool IsMono
         {
             get
             {
-                if (IsRunningInVisualStudio)
+                if (IsWindows && IsRunningInVisualStudio)
```

Comment:
How about IsRunningInVisualStudio doing the IsWindows check? 
That'd make things a bit faster for all scenarios and not just Mono.

Reply:
Done in 1248637